<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new channel...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="description_label">
     <property name="text">
      <string>Add a new TV channel into the TV-Maxe's channel list by filling the following form. Please note that TV-Maxe only supports the following streaming formats: </string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Channel name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="channel_name_lineedit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Stream address:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="stream_url_lineedit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="tv_radiobutton">
         <property name="text">
          <string>TV</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">channel_type_buttongroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_radiobutton">
         <property name="text">
          <string>Radio</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">channel_type_buttongroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Stream parameters:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTreeView" name="params_treeview">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="itemsExpandable">
          <bool>false</bool>
         </property>
         <property name="expandsOnDoubleClick">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="add_param_button">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-add">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_param_button">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset theme="list-remove">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Icon:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="select_icon_button">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset theme="document-new"/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>(GIF, PNG or SVG)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_param_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>addParameter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>remove_param_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>removeParameter()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>params_treeview</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Dialog</receiver>
   <slot>selectionChanged(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>select_icon_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>chooseChannelIcon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addParameter()</slot>
  <slot>removeParameter()</slot>
  <slot>selectionChanged(QModelIndex)</slot>
  <slot>chooseChannelIcon()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="channel_type_buttongroup"/>
 </buttongroups>
</ui>
